spec/refile/cloudinary_spec.rb

```diff
@@ -1,7 +1,34 @@
 require "spec_helper"
+require "refile/cloudinary"
+
+RSpec.describe Refile::Cloudinary do
+  let(:config) { {
+    cloud_name: "hfkkd49r3",
+    api_key: "some-key",
+    api_secret: "some-secret"
+  } }
 
-describe Refile::Cloudinary do
   it "has a version number" do
     expect(Refile::Cloudinary::VERSION).not_to be nil
   end
+  
+  describe "behavior Refile::Cloudinary" do
+    let(:backend) { Refile::Cloudinary.new( {**config} ) }
+
+    it "object not nil" do
+      expect(backend).not_to be_nil 
+    end
+
+    it "file max size 10 megabytes" do
+      expect(backend.max_size).to be <= 10_485_760
+    end
+
+    it "file great than 10 megabytes is invalid" do
+      expect(backend.max_size).to_not be > 10_485_760
+    end
+
+    it "default max_size 10 megabytes" do
+      expect(backend.max_size).to eq 10_485_760
+    end
```

Comment:
This test is fine. I'd write another test where you initialise the backend with a different value for `max_size` and test that the accessor correctly returns it.